webpack.config.js

```diff
@@ -0,0 +1,126 @@
+/**
+ * Webpack Dev Server 2.X
+extract-text-webpack-plugin 3.X <= new API surface
+ */
+
+const { DefinePlugin } = require("webpack");
+const HtmlWebpackPlugin = require("html-webpack-plugin");
+const path = require("path");
+
+function requireAll(array) {
+  // https://stackoverflow.com/a/34574630/1559386
+  return array.map(require.resolve);
+}
+
+module.exports = {
+  entry: "./src/js/index.js",
+  output: {
+    filename: "bundle.js",
+    path: path.resolve(__dirname, "dist")
+  },
+  resolve: {
+    alias: {
+      PluginSDK: path.resolve(__dirname, "src/js/plugin-bridge/PluginSDK"),
+      PluginTestUtils: path.resolve(
+        __dirname,
+        "src/js/plugin-bridge/PluginTestUtils"
+      ),
+      "#EXTERNAL_PLUGINS": path.resolve(
+        __dirname,
+        `${process.env.npm_config_externalplugins || "plugins"}`
+      ),
+      "#PLUGINS": path.resolve(__dirname, "plugins"),
+      "#SRC": path.resolve(__dirname, "src")
+    },
+    modules: [
+      // include packages
+      path.resolve(__dirname, "packages"),
+      // load dependencies of main project from within packages
+      path.resolve(__dirname, "node_modules"),
+      // load dependencies of packages themselves
+      "node_modules"
+    ]
+  },
+  node: {
+    // Jison loader fails otherwise
+    fs: "empty"
+  },
+  devtool: "cheap-module-eval-source-map",
+  devServer: {
+    // TODO: https://webpack.js.org/configuration/dev-server/#devserver-hot
+    contentBase: path.join(__dirname, "dist"),
+    open: true,
+    overlay: true,
+    port: 4200,
+    proxy: require("./webpack.proxy.dev.js")
+  },
+  plugins: [
+    new DefinePlugin({
+      "process.env": {
+        NODE_ENV: JSON.stringify("production"),
+        LATER_COV: false
+      }
+    }),
+    new HtmlWebpackPlugin({
+      template: "./src/index.html"
+    })
+  ],
+  module: {
+    rules: [
+      {
+        test: /\.js$/,
+        exclude: /node_modules/,
+        use: {
+          loader: "babel-loader",
+          options: {
+            presets: requireAll([
```

Comment:
Why not use `.babelrc`

Reply:
100% with you, but we had it like this before and I don't want to change everything in one single PR. I would suggest to move to a babelrc later on when we update babel 👍

Reply:
I mean there is a lot things being change here so adding `.babelrc` won't make big of a difference when reviewing anyways.

But at the same time I would not block the PR just because there is no `.babelrc`

Reply:
Actually there is a .babelrc file, the problem is that we're loading .js files from the external plugins folder that don't contain the babel-react e.t.c. node module.

Reply:
Yeah, we would need to load our own babelrc, parse it and stuff the output through the `requireAll` function (at least from my perspective)